content/035_module_1/_index.md

```diff
@@ -9,15 +9,15 @@ weight: 4
 
 ### Welcome
 
-In this workshop we will learn to perform a "dark release" of a new feature in our demo application using AWS App Runner.
+In this workshop we will learn to perform a feature release of new features in our demo application, while its running live on AWS App Runner. Once we learn how to create and release features, we'll leverage user targeting to influence which users can consume additonal features. This method of software release is referred to as "dark launching" and it allows developers to release code to different user cohorts without changing their expereince or redeploying the application code. 
```

Comment:
a feature release of new features -> a release of new features, or a new release